simulators/portal-interop/src/main.rs

```diff
@@ -237,574 +209,104 @@ dyn_async! {
-                    always_run: false,
-                    run: test_find_content_block_body_block_7000000,
-                    client_a: client_a.clone(),
-                    client_b: client_b.clone(),
-                }
-            ).await;
-
-            // Test find content Header With Proof
-            test.run(
-                TwoClientTestSpec {
-                    name: format!("FindContent Block Body: block number 15600000 (post-merge) {} --> {}", client_a.name, client_b.name),
-                    description: "".to_string(),
-                    always_run: false,
-                    run: test_find_content_block_body_block_15600000,
-                    client_a: client_a.clone(),
-                    client_b: client_b.clone(),
-                }
-            ).await;
-
-            // Test find content Header With Proof
-            test.run(
-                TwoClientTestSpec {
-                    name: format!("FindContent Block Body: block number 17510000 (post-shanghai) {} --> {}", client_a.name, client_b.name),
-                    description: "".to_string(),
-                    always_run: false,
-                    run: test_find_content_block_body_block_17510000,
-                    client_a: client_a.clone(),
-                    client_b: client_b.clone(),
-                }
-            ).await;
-
-            // Test find content Header With Proof
-            test.run(
-                TwoClientTestSpec {
-                    name: format!("FindContent Receipts: block number 7000000 {} --> {}", client_a.name, client_b.name),
-                    description: "".to_string(),
-                    always_run: false,
-                    run: test_find_content_receipts_block_7000000,
-                    client_a: client_a.clone(),
-                    client_b: client_b.clone(),
-                }
-            ).await;
-
-            // Test find content Header With Proof
-            test.run(
-                TwoClientTestSpec {
-                    name: format!("FindContent Receipts: block number 15600000 (post-merge) {} --> {}", client_a.name, client_b.name),
-                    description: "".to_string(),
-                    always_run: false,
-                    run: test_find_content_receipts_block_15600000,
-                    client_a: client_a.clone(),
-                    client_b: client_b.clone(),
-                }
-            ).await;
+            for ProcessedContent { content_type, block_number, test_data } in process_content(content.clone()) {
+                test.run(
+                    NClientTestSpec {
+                        name: format!("RecursiveFindContent {}: block number {}{} {} --> {}", content_type, block_number, get_flair(block_number), client_a.name, client_b.name),
+                        description: "".to_string(),
+                        always_run: false,
+                        run: test_recursive_find_content_x,
```

Comment:
Also here, I'd be extremely happy if we could just rename all of the `_x` suffix fns. Imo the `_x` suffix really isn't adding anything useful/descriptive in the fn name, and simply dropping it won't make it more difficult to understand what's going on